File: Stuck in a time Loop/src/timeloop.java
/*
Open Kattis problem Stuck in a time loop: https://open.kattis.com/problems/timeloop
Mahadi Ahmed 27-11-17
* */
public class timeloop {
  public static void main(String[] args) {
    Kattio kattio = new Kattio(System.in,System.out);
    int n = kattio.getInt();
    String abra = " Abracadabra";
    for (int i = 1; i < n + 1; i++) {
      System.out.println(i + abra);
    }
  }
}

File: Stuck in a time Loop/timeloop.cpp
/*
	Open kattis problem Stuck in a timeloop: https://open.kattis.com/problems/hissingmicrophone
	Mahadi 29-11-17
*/

#include <iostream>

int main() {
	int inputValue = 0;
	std::cin >> inputValue;
	for (int i = 1; i < inputValue + 1 ; ++i){
		std::cout << i << " Abracadabra\n";
	}
	return 0;
}